ecowitt2mqtt/helpers/publisher/mqtt/hass.py

```diff
@@ -519,6 +520,7 @@ def _get_discovery_info(
 
         if self._config.hass_entity_id_prefix:
             discovery.object_id = f"{self._config.hass_entity_id_prefix}_{payload_key}"
```

Comment:
You shouldn't be sending object_id, which is deprecated.  Suggest moving this f string to within the  default_entity_id one.